emission/core/wrapper/common_trip.py

```diff
@@ -0,0 +1,14 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class CommonTrip(ecwb.WrapperBase):
+    props = {"start_loc" : ecwb.WrapperBase.Access.WORM, # Lat/lng start point of trip
+             "end_loc" : ecwb.WrapperBase.Access.WORM, # Lat/lng end point of trip
+             "trips" : ecwb.WrapperBase.Access.WORM, # List of trip_ids that are associated with this common trip
+             "probabilites" : ecwb.WrapperBase.Access.WORM  # a matrix that represents the probabilites for edge
```

Comment:
How do I access the probability distribution for a particular field (e.g. probability distribution for start_time)? Some documentation on that would be good.